test-app/package.json

```diff
@@ -0,0 +1,125 @@
+{
+  "name": "test-app",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Test app for ember-style-modifier addon",
+  "repository": "",
+  "license": "MIT",
+  "author": "",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint:css": "stylelint \"**/*.css\"",
+    "lint:css:fix": "concurrently \"npm:lint:css -- --fix\"",
+    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint:hbs": "ember-template-lint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
+    "lint:types": "tsc --noEmit",
+    "start": "ember serve",
+    "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
+    "test:ember": "ember test"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.23.6",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/string": "^3.1.1",
+    "@ember/test-helpers": "^3.2.1",
+    "@embroider/test-setup": "^3.0.3",
+    "@glimmer/component": "^1.1.2",
+    "@glimmer/tracking": "^1.1.2",
+    "@glint/environment-ember-loose": "^1.2.2",
+    "@glint/template": "^1.2.2",
+    "@tsconfig/ember": "^3.0.2",
+    "@types/ember": "^4.0.10",
+    "@types/ember-data": "^4.4.15",
+    "@types/ember-data__adapter": "^4.0.5",
+    "@types/ember-data__model": "^4.0.4",
+    "@types/ember-data__serializer": "^4.0.5",
+    "@types/ember-data__store": "^4.0.6",
+    "@types/ember__application": "^4.0.10",
+    "@types/ember__array": "^4.0.9",
+    "@types/ember__component": "^4.0.21",
+    "@types/ember__controller": "^4.0.11",
+    "@types/ember__debug": "^4.0.7",
+    "@types/ember__destroyable": "^4.0.4",
+    "@types/ember__engine": "^4.0.10",
+    "@types/ember__error": "^4.0.5",
+    "@types/ember__helper": "^4.0.5",
+    "@types/ember__modifier": "^4.0.8",
+    "@types/ember__object": "^4.0.11",
+    "@types/ember__owner": "^4.0.8",
+    "@types/ember__polyfills": "^4.0.5",
+    "@types/ember__routing": "^4.0.19",
+    "@types/ember__runloop": "^4.0.8",
+    "@types/ember__service": "^4.0.8",
+    "@types/ember__string": "^3.16.3",
+    "@types/ember__template": "^4.0.5",
+    "@types/ember__test": "^4.0.5",
+    "@types/ember__utils": "^4.0.6",
+    "@types/qunit": "^2.19.9",
+    "@types/rsvp": "^4.0.8",
+    "@typescript-eslint/eslint-plugin": "^6.14.0",
+    "@typescript-eslint/parser": "^6.15.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "concurrently": "^8.2.2",
+    "ember-auto-import": "^2.7.0",
+    "ember-cli": "~5.5.0",
+    "ember-cli-app-version": "^6.0.1",
+    "ember-cli-babel": "^8.2.0",
+    "ember-cli-clean-css": "^3.0.0",
+    "ember-cli-dependency-checker": "^3.3.2",
+    "ember-cli-htmlbars": "^6.3.0",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-terser": "^4.0.2",
+    "ember-data": "~5.3.0",
```

Comment:
I don't think we need ember-data here.

Reply:
aggr.. added by blueprint and i have forgot to remove... sry